akka-stream/src/main/scala/akka/stream/impl/io/TcpStages.scala

```diff
@@ -178,7 +178,7 @@ import akka.util.ByteString
 
       private def unbindCompleted(): Unit = {
         stageActor.unwatch(listener)
-        unbindPromise.trySuccess(Done)
+        unbindPromise.trySuccess(())
```

Comment:
fascinating that Scala 2 allows that when reading it like this, but not so strange when thinking some more about it